azure-kusto-data/source/connectionBuilder.ts

```diff
@@ -120,44 +125,33 @@ export class KustoConnectionStringBuilder {
         return kcsb;
     }
 
-    static withAadApplicationCertificateAuthentication(connectionString: string, aadAppId: string, certificate: string, thumbprint: string, authorityId: string) {
+    static withAadApplicationCertificateAuthentication(connectionString: string, aadAppId: string, applicationCertificatePrivateKey: string, applicationCertificateThumbprint: string,  applicationCertificateX5c?: string, authorityId?: string) {
```

Comment:
Is this a breaking change?

Reply:
Moved applicationCertificateX5c to the end so it won't be a breaking change